recipes/inference/local_inference/chat_completion/chat_completion.py

```diff
@@ -75,6 +75,11 @@ def main(
 
     chats = tokenizer.apply_chat_template(dialogs)
 
+    terminators = [
```

Comment:
based on the latest changes in generation config, https://huggingface.co/meta-llama/Meta-Llama-3-8B-Instruct/discussions/4/files lets remove this part.